.github/workflows/workflow.yml

```diff
@@ -9,15 +9,15 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Might this cause issues if the latest version introduces a breaking change? Can that make our build fail without us making changes?

Reply:
I guess the only thing it would affected is Docker, as everything else is self contained inside the container.
I don't believe we rely on anything else in the OS, but not keeping it up to date is just as risky and we get warnings about that, hence why I bumped it.

Reply:
Looking at https://github.com/actions/runner-images/tree/main/images/ubuntu this shows they only use the LTS (Long Term Supported) versions of Ubuntu, these are highly stable releases so should provide a greater level of stability.